src/tools/rbd/rbd.cc

```diff
@@ -3,8 +3,30 @@
 
 #include "tools/rbd/Shell.h"
 
+#ifdef _WIN32
+
+#include <iostream>
+
+#include "common/win32/wstring.h"
+
+int main(int argc, const char **argv)
+{
+  setlocale(LC_ALL, ".UTF8");
+  SetConsoleOutputCP(CP_UTF8);
+  char** argv_utf8 = get_utf8_argv();
```

Comment:
Just wondering, instead of doing this `setlocale` + `get_utf8_argv` dance in each program, can we simply install a UTF-8 process code page at build time?  That should cause `main()` to get UTF-8 instead of ANSI:

https://learn.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page

Reply:
Good point. As far as I remember, there were some mingw toolchain limitations preventing us from using Windows app manifests but I'll take another look.